lessons/r/dplyrmagrittr/lesson.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: dplyr and magrittr
```

Comment:
Just going with what Lina had in the schedule! Can't hurt to be more informative though.